javascript/builtins/Error.json

```diff
@@ -633,6 +683,48 @@
             }
           }
         },
+        "stackTraceLimit": {
+          "__compat": {
+            "tags": [
+              "web-features:javascript"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "17"
```

Comment:
Earliest version it shows up in console in browserstack.

Reply:
```suggestion
                "version_added": "3"
```